kubernetes/cray-sysmgmt-health/values.yaml

```diff
@@ -1192,3 +1192,10 @@ smartmetrics:
     repository: artifactory.algol60.net/csm-docker/stable/quay.io/galexrt/node-exporter-smartmon
     tag: v0.1.1
     pullPolicy: IfNotPresent
+
+iscsimetrics:
+  enabled: true
```

Comment:
This is not used.  Should the `iscsi_daemon.yaml` file be conditional on this?

Reply:
This is required for prometheus to scrape data as I understand from monitoring team.

Reply:
Okay, thanks for the explanation